src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1030,15 +1034,6 @@ protected override bool CanToggleCanNavigateBackground(object parameter)
 
         internal void ComputeFrameUpdate()
         {
-#if DEBUG
-            if (renderTimer.IsRunning)
```

Comment:
Why was this deleted? I saw that control still went here some times during my debugging.

Reply:
Since we are scheduling the timer stop for after rendering occurs, this was actually disrupting the elapsed time obtained, because it used the same timer.
While we could change this to use another timer instead of deleting it, I didn't see this provide accurate results during my tests, so I just deleted it. Also, the new measure includes what this is supposed to include, so I don't think we want to track this separately.